### PR TITLE
YARN-9676. Add DEBUG and TRACE level messages to AppLogAggregatorImpl…

### PR DESCRIPTION
**Pull request by adamantal**
… and connected classes

Also removed cleanupOldLogTimes and unused methods.

Looks to me that `AggregatedLogFormat` and `LogAggregationFileController` has proper logging.

### CODE REVIEW

File: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/logaggregation/AppLogAggregatorImpl.java [hunk 1]
```diff
@@ -522,14 +538,17 @@ private void doAppLogAggregationPostCleanUp() {
         lfs.getFileStatus(logPath);
         localAppLogDirs.add(logPath);
       } catch (UnsupportedFileSystemException ue) {
-        LOG.warn("Log dir " + rootLogDir + "is an unsupported file system", ue);
+        LOG.warn("Log dir " + rootLogDir +
```
Comment (pbacsko) [inline hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/logaggregation/AppLogAggregatorImpl.java]: Use the {} placeholders

File: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/logaggregation/AppLogAggregatorImpl.java [hunk 2]
```diff
@@ -386,6 +395,13 @@ private void uploadLogsForContainers(boolean appFinished)
       if (logAggregationSucceedInThisCycle && deletionTask != null) {
         delService.delete(deletionTask);
       }
+      if (!diagnosticMessage.isEmpty()) {
+        LOG.debug("Sending log aggregation report along with the " +
+            "following diagnostic message:\"{}\"", diagnosticMessage);
+      }
+      if (!logAggregationSucceedInThisCycle) {
+        LOG.info("Log aggregation did not succeed in this cycle");
```
Comment (pbacsko) [inline hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/logaggregation/AppLogAggregatorImpl.java]: shouldn't this be a WARN or ERROR?

File: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/logaggregation/AppLogAggregatorImpl.java (hunk 1 above)
Comment (adamantal) [reply hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/logaggregation/AppLogAggregatorImpl.java]: Done

File: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/logaggregation/AppLogAggregatorImpl.java (hunk 2 above)
Comment (adamantal) [reply hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/logaggregation/AppLogAggregatorImpl.java]: Agreed. Modified accordingly.

**Comment by szilard-nemeth** [comment]
Hi @adamantal !
Thanks for this PR!
Looks good to me, +1!
Merging it in.
Will try to backport to branch-3.2 and branch-3.1, see updates in jira.

**Pull request merged**